projects.js

```diff
@@ -0,0 +1,178 @@
+    const modal = document.createElement('div');
+    document.body.style.overflow = 'hidden';
+    Workitem.append(main);
+    main.append(modal);
+    modal.className = 'modal';
+    const cancelicondesktop = new Image();
+    cancelicondesktop.src = 'images/Disabled.png';
+    cancelicondesktop.className = 'cancel-icon-desktop';
+    modal.appendChild(cancelicondesktop);
+    const cancelicon = new Image();
+    cancelicon.src = 'images/cancel.png';
+    cancelicon.className = 'cancel-icon';
+    modal.appendChild(cancelicon);
+    cancelicon.addEventListener('click', () => {
+      main.remove();
+      btndiv.appendChild(btn);
+      detaildiv.appendChild(ul);
+      detaildiv.appendChild(btndiv);
+      document.body.style.overflow = 'visible';
+    });
+    cancelicondesktop.addEventListener('click', () => {
+      main.remove();
+      btndiv.appendChild(btn);
+      detaildiv.appendChild(ul);
+      detaildiv.appendChild(btndiv);
+      document.body.style.overflow = 'visible';
+    });
+
+    const imagediv = document.createElement('div');
+    imagediv.className = 'image-div';
+    const image = new Image();
+    image.src = work.image;
+    imagediv.appendChild(image);
+    Workitem.appendChild(imagediv);
+    modal.appendChild(image);
+    image.className = 'img-project';
+    const h2 = document.createElement('h2');
+    h2.className = 'title-project';
+    h2.textContent = work.title;
+    modal.appendChild(h2);
+    const Uldetail = document.createElement('ul');
+    Uldetail.className = 'tech-stack-detail';
+    const array = work.technologies;
+    array.forEach((value) => {
+      const Lidetail = document.createElement('li');
+      Lidetail.textContent = value;
+      Uldetail.appendChild(Lidetail);
+    });
+    modal.appendChild(Uldetail);
+    const p = document.createElement('p');
+    p.className = 'description';
+    p.textContent = work.description1;
+    const p2 = document.createElement('p');
+    p2.textContent = work.description2;
+    p2.className = 'description';
+    modal.appendChild(p);
+    modal.appendChild(p2);
+    const Btndivinner = document.createElement('div');
+    Btndivinner.className = 'btndiv';
+    const source = document.createElement('a');
+    const demo = document.createElement('a');
+    demo.className = 'btn';
+    source.className = 'btn';
+    demo.textContent = 'See live';
+    source.textContent = 'See source';
+    const iconlive = new Image(20, 20);
+    iconlive.className = 'project-icons';
+    iconlive.src = 'images/live-demo.png';
+    const iconsource = new Image();
+    iconsource.className = 'project-icons';
+    iconsource.src = 'images/source.png';
+    demo.appendChild(iconlive);
+    source.appendChild(iconsource);
+    imgdiv.appendChild(img);
+    demo.setAttribute('href', work.live_version);
+    source.setAttribute('href', work.source);
+    Btndivinner.appendChild(demo);
+    Btndivinner.appendChild(source);
+    modal.appendChild(Btndivinner);
```

Comment:
- [ ] Your popup modal design is very different from the provided Figma design. Notice the correct positions of the `close icons`, the two buttons - `see live`, and `see source`, and even the text - title and paragraphs in the Figma design in comparison to your design so far. Please take time and ensure that you stick to the design as much as possible. 
- [ ] Also, please ensure that your project cards and modal popup **TEXT** are using the correct styles - `font-size',  `text-color`, 
- [ ] There should be adequate spaces on the `top`, `right`, `left`, and `bottom` around the modal as shown in the Figma image below. Please fix this

Your design
![image](https://github.com/batoolfatima2135/Portfolio/assets/84156714/0e708416-ee9d-4877-976a-c9cb5e025a2d)

Figma Design
![image](https://github.com/batoolfatima2135/Portfolio/assets/84156714/7c09f3f8-47eb-4373-ac4b-32846283425e)